src/Xamarin.Android.Build.Tasks/Tasks/GetAotArguments.cs

```diff
@@ -30,6 +27,9 @@ public class GetAotArguments : AndroidAsyncTask
 		[Required]
 		public string AndroidBinUtilsDirectory { get; set; }
 
+		[Required]
+		public string TargetName { get; set; }
+
```

Comment:
Do we need this on this class? Its not used in this class only in GetAotAssemblies

Reply:
`<Aot/>` (classic) and `<GetAotAssemblies/>` (.NET 6) both subclass `GetAotArguments`. We could move this property to exist in both of these tasks?

Reply:
Hmm. Probably best leave it as is in that case.